cmd/libs/go2idl/client-gen/generators/client_generator.go

```diff
@@ -120,7 +120,7 @@ func packageForGroup(gv unversioned.GroupVersion, typeList []*types.Type, packag
 			return generators
 		},
 		FilterFunc: func(c *generator.Context, t *types.Type) bool {
-			return types.ExtractCommentTags("+", t.SecondClosestCommentLines)["genclient"] == "true"
+			return types.ExtractCommentTags("+", t.SecondClosestCommentLines)["genclient"] != nil
```

Comment:
You're right in theory.  I can make that a little more straightforward, I
think

On Tue, Jul 5, 2016 at 4:12 AM, Wojciech Tyczynski <notifications@github.com
> wrote:

> In cmd/libs/go2idl/client-gen/generators/client_generator.go
> <https://github.com/kubernetes/kubernetes/pull/25978#discussion_r69545548>
> :
>
> > @@ -120,7 +120,7 @@ func packageForGroup(gv unversioned.GroupVersion, typeList []*types.Type, packag
> >  			return generators
> >  		},
> >  		FilterFunc: func(c *generator.Context, t *types.Type) bool {
> > -			return types.ExtractCommentTags("+", t.SecondClosestCommentLines)["genclient"] == "true"
> > +			return types.ExtractCommentTags("+", t.SecondClosestCommentLines)["genclient"] != nil
>
> I think that all changes like this one are actually incorrect, because you
> can potentially set "genclient=false" in the file and this would be treated
> as "genclient=true" after your change.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/25978/files/d4a5536797bea70a9ab71dd629fbcda0aa8d3254#r69545548>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFVgVJI_NWCciGrnfVlqETvvfr9AxGKwks5qSjwjgaJpZM4Ijc3K>
> .
>